.github/workflows/sonar-step-springboot-analyze.yml

```diff
@@ -100,6 +107,15 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      - name: Set Maven settings
```

Comment:
magari questo lo skipperei se non ci sono i dati in input

Reply:
skippa gia all'interno dell'action: https://github.com/zupit-it/pipeline-templates/pull/125/files/1ed2b694981cc3cf1a3319af50ac7ad718d78760#diff-273473d6fa3afe83f2225f9eb6849b1ad656fa0da959ee5877463986b86fe958R21